README.md

```diff
@@ -34,11 +34,15 @@
 
 ## Requirements
 
-Starting from version `0.23.2` of the library your project is required to use:
+Starting from version `0.23.3` of the library your project is required to use:
 
 * Gradle `7.0` or newer
 
-* Kotlin `1.7.0` or newer
+* Kotlin `1.9.0` or newer
```

Comment:
`## Atomicfu compiler plugin` sections seems to be obsolete as it describes how the plugin should be configured for older versions.

As there's a suggestion to use an older atomicfu version with older Kotlin version, it does make sense to move everything related to older versions to a dedicated page (or leave it here, but stress that it only applies to older atomicfu versions).

Reply:
I've fixed this section, added a note about Kotlin version requirements for the current atomicfu version and moved old configurations to the hidden section.